README.md

```diff
@@ -14,13 +14,14 @@
 [![Bugs](https://sonarqube.eea.europa.eu/api/project_badges/measure?project=volto-workflow-progress-develop&metric=bugs)](https://sonarqube.eea.europa.eu/dashboard?id=volto-workflow-progress-develop)
 [![Duplicated Lines (%)](https://sonarqube.eea.europa.eu/api/project_badges/measure?project=volto-workflow-progress-develop&metric=duplicated_lines_density)](https://sonarqube.eea.europa.eu/dashboard?id=volto-workflow-progress-develop)
 
-
 Toolbar enhanced workflow [Volto](https://github.com/plone/volto) add-on
 
 ## Features
 
 - Toolbar enhanced workflow slot
 
+![Workflow Progress](./docs/volto-workflow-progress.gif)
```

Comment:
@ana-oprea Please update path to absolute url

Reply:
@avoinea I updated the path to the gif